### DISCUSSION TITLE
How to block users on a relation with multiple restrictions

### DISCUSSION THREAD

**Discussion by keitherw** [openfga/community#377] [General]
I cannot seem to figure out how to block a user from the can_delete but adding them to a blocked relation. I am unable to add the "but not" on the can_delete: line, seems like if there is an and or and or, it will not let you.

model
  schema 1.1

type user

type document
  relations
    define owner: [organization]
    define writer: [user]
    define can_write: writer
    define can_delete: writer and member from owner

type organization
  relations
    define member: [user]

**Comment by aaguiarz** [comment]
@keitherw you can use parentheses, e.g.

`
model
   schema 1.1

type user

type document
  relations
      define owner: [organization]
      define writer: [user]
      define can_write: writer
      define blocked : [user]
      define can_delete: (writer and member from owner) but not blocked

type organization
   relations
      define member: [user]
`

Let me know if that works.

**Reply by keitherw** [reply]
Great. It worked. Thanks.
